src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const part of actions) {
+    const { type } = part;
+    const { keysToRemove } = part;
+    const { extraData } = part;
+
+    switch (type) {
+      case 'addProperties':
+        Object.assign(state, extraData);
+        break;
+
+      case 'removeProperties':
+        for (const removeKeys of keysToRemove) {
```

Comment:
```suggestion
        for (const key of keysToRemove) {
```

Reply:
> 

Is it really ok to use the same variable name in loop twice?